src/shared/reducers/ui.js

```diff
@@ -14,6 +15,7 @@ import type { UIState } from '../flow-types';
 const defaultUserState = {
     hasSeenHelpBoxType1: false,
```

Comment:
what is this referring to? the initial language selection screen?

Reply:
it's a flag set to true the first time you see the help box for project type 1 (`built_area`). the naming is not the best, indeed. I've added a short comment in the code.